src/bridge/EigenDABlobVerifierL1.sol

```diff
@@ -14,6 +18,23 @@ contract EigenDABlobVerifierL1 is IRollupManager {
         IEigenDAServiceManager.BlobHeader calldata blobHeader,
         EigenDARollupUtils.BlobVerificationProof calldata blobVerificationProof
     ) external view {
+        /*
+            Verify that the certificate is less than 2 epochs old from the L1 reference block number
```

Comment:
2 epoch is about 24mins. In addition, referenceBlockNumber is not right, because it is smaller than confirmationBlockNumber. ReferenceBlockNumber is the number we used to anchor the stake used to distribute chunks to DA nodes. The time diff is the batchingInterval+75(which is some number we introduced to prevent reorg).

Reply:
If we stick with confirmationBlockNumber. I think 20 mins is sufficient.